health-services/project-factory/src/server/utils/campaignUtils.ts

```
@@ -635,6 +765,18 @@
     delete request.body.CampaignDetails.campaignDetails
 }
 
+async function makeParentInactiveOrActive(requestBody: any, active: boolean) {
+    let parentCampaign = requestBody?.parentCampaign
```

Biome

[error] This let declares a variable that is only assigned once.

'parentCampaign' is never reassigned.

Safe fix: Use const instead.


(lint/style/useConst)